wpiutil/src/main/java/edu/wpi/first/util/struct/DynamicStruct.java

```diff
@@ -548,9 +593,12 @@ private long getFieldImpl(StructFieldDescriptor field, int arrIndex) {
       // for unsigned fields, we can simply logical shift and mask
       return (val >>> field.m_bitShift) & field.getBitMask();
     } else {
-      // to get sign extension, shift so the sign bit within the bitfield goes to the long's sign
-      // bit (also clearing all higher bits), then shift back down (also clearing all lower bits);
-      // since upper and lower bits are cleared with the shifts, the bitmask is unnecessary
+      // to get sign extension, shift so the sign bit within the bitfield goes to the
+      // long's sign
+      // bit (also clearing all higher bits), then shift back down (also clearing all
+      // lower bits);
+      // since upper and lower bits are cleared with the shifts, the bitmask is
+      // unnecessary
```

Comment:
What was the reason for this change?

Reply:
The auto formatter did it for some reason. I'll go and fix it.

Reply:
Weird, now there's a lot more formatting changes.
(By the way, my other comments are good to be marked as resolved- Unfortunately, [Github only allows the PR author and people with write access to the repository to mark conversations as resolved](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#resolving-conversations))

Reply:
Its because spotless is the most useless tool ever invented.